sdk/src/transaction/mod.rs

```diff
@@ -122,6 +122,10 @@ pub enum TransactionError {
     /// Transaction loads a writable account that cannot be written
     #[error("Transaction loads a writable account that cannot be written")]
     InvalidWritableAccount,
+
+    /// Transaction is too large
+    #[error("Transaction too large")]
+    TooLarge,
```

Comment:
I was thinking there might be other places we'd want to use this for consistency -- like rpc simulateTransaction or RpcClient -- but I guess the return types are already too different, and there's no comparable `verify_transaction()`